### PR TITLE
Added a script to autorun Update Daemon

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #214 

This now creates 2 systemd files which are placed in the respective folders

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@hongquan , I have updated the PR

File: Deploy/Systemd/ss-media-daemon.service.app [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Media Daemon
```
Comment (hongquan) [inline Deploy/Systemd/ss-media-daemon.service.app]: Why the name is still so vague? If we look into system log, how can we know what service is this? Is it of the OS or ours?

File: Deploy/auto_boot.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+SCRIPT_PATH=$(realpath $0)
+DIR_PATH=$(dirname $SCRIPT_PATH)
+
+mv $DIR_PATH/ss-media-daemon.service /lib/systemd/system/ss-media-daemon.service
```
Comment (hongquan) [inline Deploy/auto_boot.sh]: Why `mv`?

File: Deploy/auto_boot.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+mv $DIR_PATH/ss-update-daemon.service /lib/systemd/system/ss-update-daemon.service
+
+sudo systemctl daemon-reload
+sudo systemctl enable ss-media-daemon.service
+sudo systemctl enable ss-update-daemon.service
```
Comment (hongquan) [inline Deploy/auto_boot.sh]: HOW MANY TIMES I complaint you about this? You never listen???

**Comment by hongquan** [comment]
Look into

https://github.com/fossasia/susi_linux/blob/424b9cbfc4dee904fea75dae6407a2619c6840b9/update_daemon/update_daemon.sh#L21-L25

This script doesn't look like a daemon, a process that keeps running. Why do you name it "daemon" to cause confusion?

If it is not daemon, the service type (in systemd) must be changed, cannot be `Type=simple`.

**Comment by stealthanthrax** [comment]
@hongquan , updated

**Comment by hongquan** [comment]
Read the comment https://github.com/fossasia/susi_linux/pull/242#issuecomment-404380709 carefully!

**Comment by stealthanthrax** [comment]
@hongquan , updated

**Pull request merged**